tools/updateRollup/updateRollup.ts

```diff
@@ -99,17 +108,124 @@ async function main() {
         throw new Error(`Address ${deployer.address} does not have the UPDATE_ROLLUP_ROLE role`);
     }
 
-    // Add a new rollup type with timelock
-    console.log(await (await rollupManagerContract.updateRollup(rollupAddress, newRollupTypeID, upgradeData)).wait());
+    const outputsJson = [];
```

Comment:
[line 106](https://github.com/0xPolygonHermez/zkevm-contracts/pull/389/files#diff-174c8353f54d39d0de6cf8854243d9626fd57e3273254c3ec2aa11afcfcf85b8R106) --> this check is not needed for MULTISIG / TIMELOCK